File: Team_Workspace/Fatma_Ahmed/cpp_tasks/task19/task19.cpp
// task19.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include <iostream>
using namespace std;


namespace Lidar {
	int range = 100;
	void print() {
		cout << "Lidar Range :" << range << endl;

	}
}
namespace Sonar {
	int range = 5;
	void print() {
		cout << "Sonar Range :" << range << endl;

	}
}
int main()
{
	Lidar::print();
	Sonar::print();


}

File: Team_Workspace/Abdelrahman-Adel/Tasks/The Twin Sensors/task_19.py
class Lidar:
    range = 100

    @staticmethod
    def print_data():
        print(f"Lidar Range: {Lidar.range}")
class Sonar:
    range = 5

    @staticmethod
    def print_data():
        print(f"Sonar Range: {Sonar.range}")
Lidar.print_data()
Sonar.print_data()
